### PR TITLE
Victor Kalinkin html-tables (initial submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
html tables student grades

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <tr>
+        <th scope="col">Student Name</th>
+        <th scope="col">Course</th>
+        <th scope="col">Grade</th>
+      </tr>
```
Comment (scbowler) [inline html-tables/index.html:13]: Everything in the "head" of the table needs to be within a `<thead>` element

File: html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <td>HTML</td>
+        <td>9001</td>
+      </tr>
+
+      <tr>
+        <td>Linus Torvalds</td>
+        <td>Git</td>
+        <td>90</td>
+      </tr>
+
```
Comment (scbowler) [inline html-tables/index.html:44]: Everything in the "body" needs to be in a `<tbody>` element.

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple changes needed.

**Comment by vkalinkin** [comment]
html table thead and tbody

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**